library/src/main/java/com/google/maps/android/clustering/view/DefaultClusterRenderer.java

```diff
@@ -575,6 +577,15 @@ public void setAnimation(boolean animate) {
         mAnimate = animate;
     }
 
+    /**
+     * {@inheritDoc} The default duration is 300 milliseconds.
+     * @param animationDuration long: The length of the animation, in milliseconds. This value cannot be negative.
+     */
+    @Override
+    public void setAnimationDuration(long animationDuration) {
```

Comment:
One last item - can you please change all the variables here to add the `Ms` suffix? For example, `animationDurationMs`, `mAnimationDurationMs`.

Reply:
Done